pyproject.toml

```diff
@@ -7,7 +7,7 @@ authors = [
 description = "Python client library for openQA API"
 readme = "README.md"
 license = { file="COPYING" }
-requires-python = ">=3.6"
+requires-python = ">=3.8"
```

Comment:
is this really necessary? we're thinking of this as 'reluctantly drop CI on 3.6 as it's hard to keep it working', not 'actively drop support for 3.6', aren't we? this does not include any change which makes the client *actually* need 3.8 to work...

Reply:
true. updated